src/windows/tools.rs

```diff
@@ -82,7 +84,19 @@ pub unsafe fn get_drive_size(handle: HANDLE) -> u64 {
     if result == TRUE {
         *pdg.PartitionLength.QuadPart() as u64
     } else {
-        0
+        // Sometimes DeviceIoControl can fail, so try using GetDiskFreeSpaceEx as a backup
+        let mut tmp: ULARGE_INTEGER = std::mem::zeroed();
+        if GetDiskFreeSpaceExW(
```

Comment:
Is there any reason not to call `GetDiskFreeSpaceExW` all the time if it works in cases where `DeviceIoControl` doesn't?

Reply:
It's a great question.  I'm leaning towards thinking that it's always safe to call `GetDiskFreeSpaceExW`.  Especially since this very same function is used to get the free space in [`refresh()`](https://github.com/GuillaumeGomez/sysinfo/blob/8a1c545fbb471cc87beaeda5dad96005386aee7c/src/windows/disk.rs#L77-L93)

Reply:
Can you confirm it and remove the useless call if it confirms our suspicions please? :)

Reply:
Ok, I tested always using `GetDiskFreeSpaceExW` on 4 different Windows machines, and it seems to be working fine.